x/programs/rust/wasmlanche-sdk/src/types.rs

```diff
@@ -1,4 +1,5 @@
-use borsh::{BorshDeserialize, BorshSerialize};
```

Comment:
This change doesn't make sense

Reply:
I haven't found a solution to disambiguate the `<[_]>::serialize` to be from `serde::Serialize` or `borsh::BorshSerialize` so it was my workaround.

Reply:
If you have a suggestion I'd be really happy because it's not looking the best.

Reply:
Thanks for the merged branch !